common-react/@dbeaver/ui-kit/src/Select/SelectField.tsx

```diff
@@ -115,22 +124,22 @@ export function SelectField<T, ItemType extends {} = SelectItem<T>>({
 
   const selectedItem = items.find(item => getItemValue(item) === currentValue);
 
-  const displayValue = selectedItem ? (selectedRender ? selectedRender(currentValue, selectedItem) : renderItem(selectedItem)) : '';
+  const displayValue = selectedRender ? selectedRender(currentValue, selectedItem) : selectedItem ? renderItem(selectedItem) : '';
 
   return (
     <div className={clsx('dbv-kit-select-field', className)}>
-      <SelectProvider value={currentValue as any} setValue={val => handleChange(val as T)}>
+      <SelectProvider value={currentValue as any} setValue={val => handleChange(val as T)} store={store}> 
         {label && <SelectLabel className={clsx(required && 'dbv-kit-select__label--required')}>{label}</SelectLabel>}
 
-        <Select name={name} disabled={disabled}>
+        <Select name={name} disabled={disabled} required={required}>
           {displayValue}
           {arrowIcon ?? <Select.Arrow />}
         </Select>
         {description && <span className="dbv-kit-select__description">{description}</span>}
 
-        <SelectPopover gutter={4} unmountOnHide>
+        <SelectPopover autoFocusOnShow={autoFocusItemsOnShow} gutter={4} unmountOnHide>
           {items.length === 0 ? (
-            <div className="dbv-kit-select__empty">No items</div>
+            <div className="dbv-kit-select__empty">{noItemsPlaceholder}</div>
```

Comment:
this prop is not required, do we need a fallback here? Or maybe make it required to define with l10n on the usage level